excitebike/src/test/java/com/codenjoy/dojo/excitebike/model/PlayersSpawnSystemParametrizedTest.java

```diff
@@ -0,0 +1,217 @@
+ * %%
+ * Copyright (C) 2018 - 2019 Codenjoy
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/gpl-3.0.html>.
+ * #L%
+ */
+
+import com.codenjoy.dojo.excitebike.model.items.bike.Bike;
+import com.codenjoy.dojo.excitebike.model.items.bike.BikeType;
+import com.codenjoy.dojo.excitebike.services.parse.MapParser;
+import com.codenjoy.dojo.excitebike.services.parse.MapParserImpl;
+import com.codenjoy.dojo.services.Dice;
+import com.codenjoy.dojo.services.EventListener;
+import com.codenjoy.dojo.services.Game;
+import com.codenjoy.dojo.services.multiplayer.Single;
+import com.codenjoy.dojo.services.printer.PrinterFactory;
+import com.codenjoy.dojo.services.printer.PrinterFactoryImpl;
+import com.google.common.collect.Lists;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@RunWith(Parameterized.class)
+public class PlayersSpawnSystemParametrizedTest {
+
+    private String name;
+    private String init;
+    private String expected;
+    private int newPlayerNumberAfterInit;
+
+    public PlayersSpawnSystemParametrizedTest(String name, int newPlayerNumberAfterInit, String init, String expected) {
+        this.name = name;
+        this.newPlayerNumberAfterInit = newPlayerNumberAfterInit;
+        this.init = init;
+        this.expected = expected;
+    }
+
+    @Parameterized.Parameters(name = "{0}")
+    public static Collection<Object> data() {
+        return Lists.newArrayList(
+                new Object[]{"shouldAddThreeBikesInFirstChessColumn",
+                        0,
+                        "■■■■■■■" +
+                                "       " +
+                                "       " +
+                                "       " +
+                                "       " +
+                                "       " +
+                                "■■■■■■■",
+                        "■■■■■■■\n" +
+                                "       \n" +
+                                "       \n" +
+                                " e     \n" +
```

Comment:
But it's not the first column, it starts from second column, isn't it?
Here and in every other case.
Could we fix it?